_posts/2019-10-01-quarkus-news-1.adoc

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: 'Quarkus News #1'
```

Comment:
@maxandersen do you think people will get what it is about? It's not about what's new in Quarkus. More about articles people write.

Reply:
I grok your concern - I went for this as it was short and sweet...others I considered was "weekly" (i.e. "Docker Weekly") but didn't want to commit to that kind of schedule (yet :)

Jboss has has "weekly editorial" ....I guess "Quarkus Editorial" would work - just sounds so...dry.

any ideas ?

Reply:
Quarkus Gazette ?  Qazette ?

Reply:
QNews ? (since QTips are used already ?)

Reply:
FWIW, I've always considered myself more of a hashmap than a linked list.

Reply:
Per above, perhaps replace Quarkus with Q:
Quarkus Report
Quarkus News
Quarkus Insider
Quarkus Times

QZone
InfoQ. Wait, nevermind :-)

Reply:
this was published earlier with "Quarkus Newsletter #1" :)